README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
Jorge, You are right about 40s delay time, after server time matching with client time, now it's working fine.
From the introduction about  [clock](http://wiki.ros.org/Clock) , If we do not synchronize the wall-clocks of multiple machines, they will not agree on temporal calculations like those used in tf. 
This is my understanding and guessing, correct me if I am wrong.
1.Now industrial_robot_simulator works at client side, then it will sends joint_states to service side robot_state_publisher to generate tf, but the time-stamp server received is client time, which doesn't match with server time, then maybe robot_state_publisher will use local server time on tf, which causes the delay.

Instead of letting all the machine synchronize their local time, maybe we need to add /clock topic to publish "simulation time" to synchronize all the clients with server. if you agree, I can plan to work at clock synchronize function in next PR along with launching process on demand function.